### PR TITLE
Feature/docs refresh

### PR DESCRIPTION
**Pull request by SamC-Apadmi**

### CODE REVIEW

File: docs/docs/desktop/overview.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Mockzilla Desktop
+
+The Mockzilla desktop app can be downloaded [here](https://mockzilla-test.pages.dev/).
+
+It allows manipulation of the mock data being returned by Mockzilla on your device.
+
+!!! important
```
Comment (Skeletonxf) [inline docs/docs/desktop/overview.md:7]: This might be worth mentioning on the initial screen of the UI too

File: docs/docs/desktop/overview.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### (1): Connect your device
+
+Your device should be automatically discovered by Mockzilla (you may need to restart the app on your device).
+
+
+![alt text](img/device_connection.png "Device connection")
+
+!!! note
+    If network discovery does find your device you can manually type in the IP address of your device. (Don't forget the port!)
```
Comment (Skeletonxf) [inline docs/docs/desktop/overview.md:19]: Should this say 'doesn't find your device'?

Comment (Skeletonxf) [reply docs/docs/desktop/overview.md:19]: Also I've been able to connect to an emulator by typing `:60000` without giving the port

**Review by Skeletonxf — APPROVED** [review]

File: docs/docs/desktop/overview.md (hunk 2 above)
Comment (SamC-Apadmi) [reply docs/docs/desktop/overview.md:19]: Yeah I'll fix that typo 🤦 . I'm guessing the only reason that worked is that you've already got port forwarding setup for that emulator from using the website. The new UI decides the port dynamically that gets used for forwarding and doesn't surface that to the UI.

**Pull request merged**